libsolidity/analysis/TypeChecker.h

```diff
@@ -47,10 +47,10 @@ class TypeChecker: private ASTConstVisitor
 {
 public:
 	/// @param _errorReporter provides the error logging functionality.
-	TypeChecker(langutil::EVMVersion _evmVersion, langutil::ErrorReporter& _errorReporter):
-		m_evmVersion(_evmVersion),
-		m_errorReporter(_errorReporter)
-	{}
+	TypeChecker(langutil::EVMVersion _evmVersion, langutil::ErrorReporter& _errorReporter)
+		: m_evmVersion(_evmVersion), m_errorReporter(_errorReporter)
+	{
+	}
```

Comment:
How would this look like with more members? Here, again my concern is that with moving the colon to the new line we'll have to always align them with spaces.